pkg/karmadactl/cmdinit/cmdinit.go

```diff
@@ -139,7 +139,7 @@ func NewCmdInit(parentCommand string) *cobra.Command {
 	flags.Int32VarP(&opts.KarmadaWebhookReplicas, "karmada-webhook-replicas", "", 1, "Karmada webhook replica set")
 	flags.StringVarP(&opts.KarmadaAggregatedAPIServerImage, "karmada-aggregated-apiserver-image", "", kubernetes.DefaultKarmadaAggregatedAPIServerImage, "Karmada aggregated apiserver image")
 	flags.Int32VarP(&opts.KarmadaAggregatedAPIServerReplicas, "karmada-aggregated-apiserver-replicas", "", 1, "Karmada aggregated apiserver replica set")
-
+	flags.IntVarP(&opts.WaitComponentReadyTimeout, "wait-component-ready-timeout", "", 30, "Wait for karmada component ready timeout")
```

Comment:
We shouldn't change the default timeout during introducing the flag.

Reply:
I've taken @lonelyCZ's advice
https://github.com/karmada-io/karmada/pull/3453#discussion_r1191947499

Reply:
I guess that's just an example of using a literal value rather than a variable.

Reply:
In fact, I think both can be used, and it is good to use the default value uniformly